eox_nelp/edxapp_wrapper/course_creators.py

```diff
@@ -0,0 +1,15 @@
+"""Wrapper course_creator module file.
+This contains all the required dependencies from course_creators.
+Attributes:
+    backend:Imported ccx module by using the plugin settings.
+    CourseCreator: Wrapper courseCreator model.
```

Comment:
```suggestion
    CourseCreator: Wrapper CourseCreator model.
```

Reply:
Resolved in https://github.com/eduNEXT/eox-nelp/pull/2/commits/a125a7569bbfd536f28897a1a2c94aa41e39a324